package.json

```diff
@@ -24,6 +24,7 @@
     "test": "test"
   },
   "scripts": {
+    "prepare": "beemo create-config --silent",
```

Comment:
What is this for?

Reply:
For making beemo init config when you haven't run any of the script.  So you have all the config for vscode right when you run yarn (`prepare` = `postinstall`)